doc/run_settings.rst

```diff
@@ -0,0 +1,159 @@
+************
+Run Settings
+************
+
+========
+Overview
+========
+SmartSim allows you to configure the run settings of a job (``Model`` or ``Ensemble``)
+through a ``RunSettings`` object. Each SmartSim `launcher` interfaces with a number of
+``RunSettings`` objects. For the local `launcher`, SmartSim supports
+a base ``RunSettings`` object. For the WLM `launchers`, SmartSim supports a variety of
+``RunSettings`` child classes.
+
+* Navigate to :ref:`local section<Local>` to configure run settings locally
+* Navigate to :ref:`wlm section<HPC>` to configure run settings for a HPC
+
+A run settings object is initialized through the ``Experiment.create_run_settings()`` function.
+This function accepts a `run_command` argument: the command to run the executable.
+If `run_command` is set to `auto`, SmartSim will attempt to match a run command on the
+system with a ``RunSettings`` class. If found, the class corresponding to
+that `run_command` will be created and returned.
+If the `run_command` is passed a recognized run command (ex. 'Slurm') the ``RunSettings``
```

Comment:
"e.g." is "for example"
"i.e." is "that is"
ex is... incorrect! 
a valid example for slurm would be "srun" or "sbatch"